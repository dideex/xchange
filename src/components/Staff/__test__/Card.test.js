import React from 'react'
import Component from '../Card'
import {shallow} from 'enzyme'

describe('Staff card', () => {
  const props = {
    photo: 'fake photo',
    name: 'fake name',
    role: 'fake role',
    svg: 'fake svg'
  }
  describe('Markup', () => {
    it('Basic markup', () => {
      const wrapper = shallow(<Component {...props} />)

      expect(wrapper.html()).toMatchSnapshot()
    })
  })
})