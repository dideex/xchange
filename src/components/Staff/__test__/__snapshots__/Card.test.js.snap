// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Staff card Markup Basic markup 1`] = `"<article class=\\"css-e6aqxp\\"><img src=\\"fake photo\\" alt=\\"Avatr\\" class=\\"css-1t6ic23\\"/><h3>fake name</h3><p>fake role</p></article>"`;
